packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -617,3 +617,25 @@ if (project.enableBatchBuilds()) {
   console.log('Batch builds were enabled');
 }
 ```
+
+## Timeouts
+
+There are two types of timeouts that can be set when creating your Project. The `timeout` property can be used to set an upper limit on how long your Project is able to run without being marked as completed. The default is 60 minutes. An example of overriding the default follows.
```

Comment:
Can you split this paragraph into multiple lines on sentence boundaries? These files should be readable in their plain-text form as well as after being rendered by a Markdown engine.

```sugguestion
There are two types of timeouts that can be set when creating your Project.
The `timeout` property can be used to set an upper limit on how long your Project is able to run without being marked as failed.
The default is 60 minutes.
An example of overriding the default:
```